apps/sms/views/conversation/js/conversation.js

```diff
@@ -2303,7 +2303,6 @@ var ConversationView = {
 
     // Clean composer fields (this lock any repeated click in 'send' button)
     this.disableConvertNoticeBanners();
-    this.cleanFields();
     this.enableConvertNoticeBanners();
```

Comment:
I see we can remove the calls to disableConvertNoticeBanners and enableConvertNoticeBanners (and the comment).

But I also realize we still need to clean the fields when we receive the event "sending" (inhttps://github.com/mozilla-b2g/gaia/blob/e6994410dcc88bfead0e44620a065220b7f12290/apps/sms/views/conversation/js/conversation.js#L941).

and even with this, I see that now we can double click on the "send" button and send the message twice. We need to prevent this by locking the composer and unlocking it after some time.

To do this, we need to setup a "canSend" function when calling "setupLock" (in both places). This function would return "true" most of the time, and "false" after sending until we get an error or the "sending" event.